lib/js/components/Cards/CardExpandable.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+	<ds-card class="cardExpandable">
+		<template #header><slot name="header"></slot></template>
+		<template #content>
+			<slot name="content" :isExpanded="isExpandedInternal" />
+			<slot v-if="isExpandedInternal" name="expandedContent" />
+		</template>
+		<template v-if="isExpanderVisible" #footer>
+			<div v-ripple class="cardExpandable__expander" @click="onExpanderClick">
+				<span class="cardExpandable__expanderLabel">{{ computedExpanderText }}</span>
+				<ds-icon
+					class="cardExpandable__expanderIcon"
+					:icon="ICONS.FA_CHEVRON_DOWN"
+					:size="ICON_SIZES.X_SMALL"
+					:rotation="isExpandedInternal ? 180 : null"
+				/>
+			</div>
+		</template>
+	</ds-card>
+</template>
+
+<style lang="scss" scoped>
+@import '../../../styles/settings/colors/tokens';
+@import '../../../styles/settings/spacings';
+@import '../../../styles/settings/typography';
+
+.cardExpandable {
+	&__expander {
+		align-items: center;
+		background-color: $neutral-background;
+		display: flex;
+		height: 44px;
+		justify-content: center;
+		padding: $space-xxs $space-s;
+
+		&:hover {
+			cursor: pointer;
```

Comment:
`cursor` will show up only on `hover` so imo it can be moved up into the `$__expander` class

Reply:
[`a96a7c6` (#103)](https://github.com/bethinkpl/design-system/pull/103/commits/a96a7c61c1ab3ab8568c970e5bd45ca9a121b546)